tests/components/scene/pool.test.js

```diff
@@ -0,0 +1,80 @@
+/* global assert, process, setup, suite, test */
+
+suite('pool', function () {
+  setup(function (done) {
+    var el = this.sceneEl = document.createElement('a-scene');
```

Comment:
You can do: 

```js
var sceneEl = helpers.entityFactory().sceneEl;
var mixinEl = helpers.mixinFactory('test', {material: 'color: red'}, sceneEl);
```

This will create `<a-assets>` and `<a-mixin>` for you.